miprometheus/models/model.py

```diff
@@ -317,38 +317,35 @@ def plot(self, data_dict, predictions, sample=0):
 
         """
 
-    def save(self, model_dir, stats):
+    def save(self, model_dir, training_status, stats):
         """
         Generic method saving the model parameters to file. It can be \
         overloaded if one needs more control.
 
         :param model_dir: Directory where the model will be saved.
         :type model_dir: str
 
-        :param stats: Statistics value to save with the model.
-        :type stats: ``StatisticsCollector`` or ``StatisticsAggregator``
+        :param training_status: String representing the current status of training.
+        :type training_status: str
+
+        :param stats: Statistics that will be saved to checkpoint along with the model.
+        :type stats: ``StatisticsAggregator``
```

Comment:
If you want to specify that the type of `params` is the `StatisticsAggregator` class, can you please use the following ? This will allow inter-linking in the doc :slightly_smiling_face: 

> :type stats: :py:class:`miprometheus.utils.StatisticsAggregator`

Reply:
👌